app/controllers/catalog_controller.rb

```diff
@@ -114,6 +114,12 @@ class CatalogController < ApplicationController
     # common method since search results and reports all do the same configuration
     add_common_date_facet_fields_to_config! config
 
+    config.add_facet_field SolrDocument::FIELD_CONSTITUENTS, label: 'Virtual Objects', home: false,
```

Comment:
I'm not sure if `Virtual Objects` is the best label here. Defer to @andrewjbtw

Reply:
This is probably getting ahead of things but if we were to start indexing other special case objects, I could see using a more generic title for the facet. The other special case I can think of are digital serials, which are items that share a title and catkey but have their own druids, like issues of the same magazine.

In that scenario you could imagine a facet header with some name to be determined and then under it you'd see:

- Virtual Object
- Digital Serial

but that might just be making more work for ourselves at this stage.

Reply:
These labels are super easy to change with very little in the way of consequences, so whatever works for both labels right now is fine by me!